userspace/libsinsp/filterchecks.cpp

```diff
@@ -80,6 +80,10 @@ int32_t gmt2local(time_t t)
 	gmt = &sgmt;
 	*gmt = *gmtime(&t);
 	loc = localtime(&t);
+	if(loc == NULL)
+	{
+		throw sinsp_exception("cannot get localtime");
```

Comment:
Can we have a proper fallback here?
Like, just return 0, i don't, and manage the issue in the `sinsp_utils::ts_to_string`.
NOTE: a line above we are de-referencing a possible NULL ptr:
```
*gmtime(&t);
```
Please fix it too.

Reply:
Looks like this is only used to check with `(0)`, on failure we probably have bigger problems.

Re. gmtime() call, done!